app/helpers/setTheme.js

```diff
@@ -0,0 +1,25 @@
+import { helper } from '@ember/component/helper';
+import setCookie from './setCookie';
+
+function setTheme() {
+  const allCookies = document.cookie.split(';');
+  let obj = {};
+  for (let cookie of allCookies) {
+    const [key, value] = cookie.split('=');
+    obj = { ...obj, [key.trim()]: value };
```

Comment:
Not very clear with what's this doing
Maybe simplify the code? Or at least add comment as to what's it doing?